design/templates/solution_design.md

```diff
@@ -67,5 +71,41 @@
 ## Open questions
 [//]: # (Add any question that is still open. It makes it easier for the reader to have the open questions accumulated here istead of them being acattered along the doc)
 
-## Implementation plan
+## Delivery plan
 [//]: # (Break the solution into tasks)
+
+### Total
+[//]: # (Write the total estimation)
```

Comment:
shouldn't this go in the end? awkward UX that i scroll through this, write the other sections and then go back

Reply:
From the reader perspective i like it at the start, but it is not critical, lets wait to other comments